server/scsynth/scsynth_main.cpp

```diff
@@ -340,6 +376,10 @@ int scsynth_main(int argc, char** argv) {
         }
     }
 
+#ifdef _WIN32
+    scsynthHasBooted();
```

Comment:
this is very close to correct, you just need to call `join` on the thread after this. if you don't `join` or `detach` a valid thread, `std::thread`'s destructor will call `std::terminate` which will effectively make the program crash, or at least appear to have 'abnormally terminated'.

Reply:
Thanks for your detailed review, I really appreciate it.
I did not know where to put the join but now that you show it, it makes sense :)